### DISCUSSION TITLE
Ability to increase spacing before headings by default

### DISCUSSION THREAD

**Discussion by EduNetEurope** [outline/outline#6384] [Ideas]
It would be amazing if we did not have to create spacings by empty paragraphs but could set a default value for spacings before headings 1, 2, 3 and so on.

**Comment by EduNetEurope** [comment]
The problem with the workaround of empty paragraphs is that they disappear when a section before the given heading is collapsed.

**Comment by EduNetEurope** [comment]
Please merge: https://github.com/outline/outline/discussions/6076

**Reply by tommoor** [reply]
A similar change was made, the heading sizing was adjusted in a recent release.

**Reply by EduNetEurope** [reply]
Would it be possible to allow us to define "spacing before" headings, notices, etc?
We would like to increase the spacing to improve the readability.

We don't think the adjustment of the heading size was enough to improve readability.
